src/GovUk.Education.ExploreEducationStatistics.Content.Model/Database/ApplicationDbContext.cs

```diff
@@ -802,7 +802,8 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
                   Title = "School and pupils and their characteristics",
                   Summary = "",
                   TopicId = new Guid("e50ba9fd-9f19-458c-aceb-4422f0c7d1ba"),
-                  Slug = "school-pupils-and-their-characteristics"
+                  Slug = "school-pupils-and-their-characteristics",
+                  LegacyPublicationUrl = new Uri("https://www.gov.uk/government/collections/statistics-school-and-pupil-numbers")
```

Comment:
The release for this publication will want removing if its no longer one of the unmoderated test releases

Reply:
I've just left it commented out for now, in case we want it back in the future